config/locales/de.yml

```diff
@@ -64,8 +64,8 @@ de:
       api_keys:
       name:
       scopes:
-      age:
       gem:
+      age:
```

Comment:
Why was this and other keys moved?

Reply:
Unless I'm missing something, It would be great to reorganize locales in the main branch - https://github.com/rubygems/rubygems.org/pull/3134  so the tool only adds new keys for new contributions